package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "test": "c8 mocha",
-    "clean": "rm -fr ./js/dist",
+    "clean": "rimraf ./js/dist/* --glob",
```

Comment:
The `rm` not working on Windows lead me down a `build:dev` path.  This will let me run `npm run build` and not confuse js/dist files.

Reply:
You're most likely the first Windows user to try building this project.

Reply:
Often 😄